### PR TITLE
Add support for fastp length_required

### PR DESCRIPTION
**Pull request by LouisLeNezet**
## PR checklist

Add support for the `length_required` option in fastp.

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Review by FriederikeHanssen — CHANGES_REQUESTED** [review]
Can you add this to the changelog and the documentation? Is this the default value for fastp?

**Comment by LouisLeNezet** [comment]
I will update it as well as the test.yml.
15 is effectively the default value of fastp
https://open.bioqueue.org/home/knowledge/showKnowledge/sig/fastp

**Comment by LouisLeNezet** [comment]
Should I add some documentation in the `output.md` ?

**Comment by FriederikeHanssen** [comment]
I think if it is the default value, than adding that as a note to the params description is fine (so people don't think we changed the default behavior massively). in general, yes adding a note to the fastp section in the output would be good. Something like "contains reads passing the fastp filtering steps for length filtering, ... " etc

File: docs/output.md [hunk 1]
```diff
@@ -41,8 +42,8 @@ The pipeline is built using [Nextflow](https://www.nextflow.io/) and processes d
       - [Sentieon DNAscope joint germline variant calling](#sentieon-dnascope-joint-germline-variant-calling)
     - [Sentieon Haplotyper](#sentieon-haplotyper)
       - [Sentieon Haplotyper joint germline variant calling](#sentieon-haplotyper-joint-germline-variant-calling)
-    - [Lofreq](#lofreq)
     - [Strelka](#strelka)
+    - [Lofreq](#lofreq)
```
Comment (maxulysse) [inline docs/output.md:46]: why changing the order of this?

Comment (LouisLeNezet) [reply docs/output.md:46]: In fact I didn't changed this line, this have been done automatically.
But it does reflect that the Strelka paragraph is before Lofreq.

Comment (FriederikeHanssen) [reply docs/output.md:46]: I think it's a markdown linting thing

**Review by FriederikeHanssen — APPROVED** [review]
Nice one!

**Pull request merged**